packages/toc/src/generators/notebook/get_markdown_heading.ts

```diff
@@ -33,7 +33,8 @@ function getMarkdownHeadings(
   dict: any,
   lastLevel: number,
   cellRef: Cell,
-  index: number = -1
+  index: number = -1,
+  isRunning = RunningStatus.Idle
```

Comment:
If we are back-porting to 3.3, should this one become optional to avoid breaking changes? It looks like it's public.